packages/shared/lib/sdk/sync.ts

```diff
@@ -392,6 +392,7 @@ export interface NangoProps {
     runnerFlags: RunnerFlags;
     debug: boolean;
     startedAt: Date;
+    endUser: { id: number; endUserId: string | null; orgId: string | null } | null;
```

Comment:
why not `endUser: EndUser | null`

Reply:
Because the schema will change at some point and I don't need anything else, plus it's passing through zod which would be annoying to verify. 
We do the same for team (and environment but differently)